images/nginx/rootfs/build.sh

```diff
@@ -151,6 +151,18 @@ get_src()
   tar xzf "$f"
   rm -rf "$f"
 }
+OPENSSLEXTRAPKG=""
+if [[ -z ${ARCH} || ${ARCH} == "x86_64" || ${ARCH} == "aarch64" ]]; then
+        wget 'http://openresty.org/package/admin@openresty.com-5ea678a6.rsa.pub'
+        mv 'admin@openresty.com-5ea678a6.rsa.pub' /etc/apk/keys/
+        . /etc/os-release
+        MAJOR_VER=`echo $VERSION_ID | sed 's/\.[0-9]\+$//'`
+
+        echo "http://openresty.org/package/alpine/v$MAJOR_VER/main" \
+                | tee -a /etc/apk/repositories
```

Comment:
```suggestion
                >> /etc/apk/repositories
```

Is `tee` used here instead of appending directly to see the echo in the terminal?

Reply:
I copied, posted, never reviewed! Sounds good not using tee :D